dipy/reconst/dki.py

```diff
@@ -359,6 +361,75 @@ def _F2m(a, b, c):
     return F2
 
 
+def _directional_kurtosis(dt, MD, kt, V, min_diffusivity=0, min_kurtosis=-1):
```

Comment:
Might this be useful as a public function, rather than a private helper function?